pkg/install/v1alpha1/snapshot_promoter_sc.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// SnapshotPromoterSCArtifacts returns the snapshot(clone) provisioner related artifacts
+func SnapshotPromoterSCArtifacts() (list ArtifactList) {
+	list.Items = append(list.Items, ParseArtifactListFromMultipleYamls(snapshotPromoterSCYaml)...)
+	return
+}
+
+// snapshotPromoterSCYaml returns all the yamls related to 
+// snapshot promote (clone) provisioner SC in a string
+// format
+//
+// NOTE:
+//  This is an implementation of MultiYamlFetcher
+func snapshotPromoterSCYaml() string {
```

Comment:
Do we need a version suffix to function name?

Reply:
same as above.